src/NodeAutoCompleteViewExtension/ViewModels/NodeAutoCompleteBarViewModel.cs

```diff
@@ -191,13 +191,20 @@ private void ReAddNode(int index)
         internal void ConsolidateTransientNodes()
         {
             var node = PortViewModel.NodeViewModel;
+
+            //unlock undo/redo
+            node.WorkspaceViewModel.Model.UndoRedoLocked = false;
```

Comment:
This is fine for now, we could also make it more granular in the future, similar to how undorecorder that the period of the lock is maintained by a living cycle of an object instead of bool setting by the coder. Something to think about

Reply:
Let me take a look at this. I think I might have some ideas.